chapters/ne/chapter0/1.mdx

```diff
@@ -0,0 +1,117 @@
+
+```
+python -m venv .env                                                                                                                                                                                                
+```
+
+
+अब तपाईंको खाली फोल्डरमा *.env* नामको डाइरेक्टरी हुनुपर्छ:                                                                                                                                                                         
+
+```
+ls -a                                                                                                                                                                                                              
+```
+
+
+```out                                                                                                                                                                                                           
+.      ..    .env                                                                                                                                                                                                 
+```                                                                                                                                                                                                                 
+
+तपाईं activate र deactivate स्क्रिप्टहरू प्रयोग गरेर भर्चुअल इन्भाइरोमेन्टमा प्रवेश र बाहिर निस्कन सक्नुहुन्छ:                                                                                                                                 
+
+
+# भर्चुअल इन्भाइरोमेन्ट सक्रिय गर्न     
+
+```                                                                                                                                                                                    
+source .env/bin/activate                                                                                                                                                                                          
+```                                                                                                                                                                                                                  
+# भर्चुअल इन्भाइरोमेन्ट निष्क्रिय गर्न  
+
+```                                                                                                                                                                                      
+deactivate                                                                                                                                                                                                        
+```                                                                                                                                                                                                                
```

Comment:
This should just be one code cell and the comments should be right before the actual command